website/profile/views.py

```diff
@@ -393,6 +394,37 @@ def user_notifications(auth, **kwargs):
         'mailing_lists': auth.user.mailing_lists
     }
 
+@dev_only
+@must_be_logged_in
+def oauth_application_list(auth, **kwargs):
+    """Return app creation page with list of known apps"""
+    app_list_url = api_v2_url("users/{}/applications/".format(auth.user._id))
+    return {
+        "app_list_url": json.dumps(app_list_url)
+    }
+
+@dev_only
+@must_be_logged_in
+def oauth_application_register(auth, **kwargs):
+    """Register an API application: blank form view"""
+    submit_url = api_v2_url("users/{}/applications/".format(auth.user._id))  # POST request to this url
+    return {"submit_url": json.dumps(submit_url),
```

Comment:
Yeah, this needs to be brought up to spec with current best practices. (see conversion scheme here: https://github.com/CenterForOpenScience/osf.io/issues/2933#issuecomment-119757673 ) 

I'd have preferred to have less data transformation going on in templates, but... meh.